README.md
# Portfolio
#Turn.js Library



changelog.txt
Turn.js
www.turnjs.com

------------------------------------------------------
Turn.js 4.1.0 - 2012/11/65
------------------------------------------------------

+ Added support to the Android's default browser
+ Fixed the hard page effect in IE9
+ Fixed issue #220

------------------------------------------------------
Turn.js 4.0.9 - 2012/08/03
------------------------------------------------------

+ Fixed issue when using the .own-size class
+ Event zoom.change now is preventable

------------------------------------------------------
Turn.js 4.0.8 - 2012/07/16
------------------------------------------------------

+ Added option direction
+ Added method direction
+ Added property direction

------------------------------------------------------
Turn.js 4.0.7 - 2012/07/12
------------------------------------------------------

+ Fixed a issue of the pages method
+ Fixed issue #112
+ corner argument of the start event now returns null when using the methods next, previous and page
+ Fixed issue of certain samples in IE8

------------------------------------------------------
Turn.js 4.0.6 - 2012/07/10
------------------------------------------------------

+ Fixed a issue of the peel method when using the single display
+ Fixed issue of using all the corners in single display mode
+ Added method version
+ Added option turnCorner

------------------------------------------------------
Turn.js 4.0.5 - 2012/06/16
------------------------------------------------------

+ Fixed some visualization issues of the zoom viewport

------------------------------------------------------
Turn.js 4.0.4 - 2012/06/16
------------------------------------------------------

+ Fixed a problem of the method page and event start

------------------------------------------------------
Turn.js 4.0.3 - 2012/06/16
------------------------------------------------------

+ Fixed minor issues of HTML4 version

------------------------------------------------------
Turn.js 4.0.2 - 2012/06/16
------------------------------------------------------

+ New Zoom viewport
+ New magazine sample

------------------------------------------------------
Turn.js 4.0.1 - 2012/06/12
------------------------------------------------------

+ Fixed some issues when calculating z-index of pages
+ Added method peel

------------------------------------------------------
Release 4 Commercial - 2012/06/07
------------------------------------------------------

+ Added option autoCenter
+ Added option zoom
+ Added property animating
+ Added property zoom
+ Added method center
+ Added method destroy
+ Added method is
+ Added method zoom
+ Added event missing
+ Added event zooming
+ Added class .even
+ Added class .fixed
+ Added class .hard
+ Added class .odd
+ Added class .own-size
+ Added class .sheet
+ Added class .sheet
+ Added the ignore attribute
+ New turn.html4.js
+ New scissors.js
+ Changed the class .turn-page to .page
+ Improved the animation frame generator with requestAnimationFrame
+ Improved the animation speed for hard pages with CSS3 transitions
+ Redesigned the event sequence to listen to only three events
+ Fixed issue #79
+ Fixed issue #91
+ Fixed issue about the event order turning
+ turned
+ Fixed issue about appending pages in wrong locations

------------------------------------------------------
Release 3 - 2012/03/01
------------------------------------------------------

+ Added 'range'
+ Added 'addPage'
+ Added 'removePage'
+ Added 'hasPage'
+ Added 'pages'
+ Added 'display'
+ Added 'when' to the initial configuration
+ Added 'pages' to the initial configuration
+ Added 'inclination' to the initial configuration
+ Added 'first' event
+ Added 'last' event
+ Added gradients for non-webkit browsers

------------------------------------------------------
Release 2 - 2012/02/15
------------------------------------------------------

+ Added 'size'.
+ Bug in Chrome 17-18 Beta about losing background-image was fixed.

------------------------------------------------------
Release 1 - 2012/02/05
------------------------------------------------------
+ First alpha release 

/* Basic sample 

body{
	overflow:hidden;
	
background-color:#fcfcfc;
	margin:0;
	padding:0;
}

.flipbook-viewport{
	overflow:hidden;
	width:100%;
	height:100%;
}

.flipbook-viewport .container{
	position:absolute;
	top:50%;
	left:50%;
	margin:auto;
}

.flipbook-viewport .flipbook{
	width:922px;
	height:600px;
	left:-461px;
	top:-300px;
}

.flipbook-viewport .page{
	width:461px;
	height:600px;
	background-color:white;
	background-repeat:no-repeat;
	background-size:100% 100%;
}

.flipbook .page{
	-webkit-box-shadow:0 0 20px rgba(0,0,0,0.2);
	-moz-box-shadow:0 0 20px rgba(0,0,0,0.2);
	-ms-box-shadow:0 0 20px rgba(0,0,0,0.2);
	-o-box-shadow:0 0 20px rgba(0,0,0,0.2);
	box-shadow:0 0 20px rgba(0,0,0,0.2);
}

.flipbook-viewport .page img{
	-webkit-touch-callout: none;
	-webkit-user-select: none;
	-khtml-user-select: none;
	-moz-user-select: none;
	-ms-user-select: none;
	user-select: none;
	margin:0;
}

.flipbook-viewport .shadow{
	-webkit-transition: -webkit-box-shadow 0.5s;
	-moz-transition: -moz-box-shadow 0.5s;
	-o-transition: -webkit-box-shadow 0.5s;
	-ms-transition: -ms-box-shadow 0.5s;

	-webkit-box-shadow:0 0 20px #ccc;
	-moz-box-shadow:0 0 20px #ccc;
	-o-box-shadow:0 0 20px #ccc;
	-ms-box-shadow:0 0 20px #ccc;
	box-shadow:0 0 20px #ccc;
}

OLD ONE
.flipbook-viewport{
	position: relative;
	border: 5px solid green;
	width: 100%;
	height: 100%;
	
	

}

.flipbook-viewport .container{
		border: 5px solid purple;
	width: 50%;
	height: 50%;
		

}

.flipbook-viewport .flipbook{
	border: 5px solid brown;
	margin-left: auto;
	margin-right: auto;
	left: 30%;
	

}

.flipbook-viewport .page{
	width:461px;
	height:600px;
	background-color:white;
	background-repeat:no-repeat;
	
}

.flipbook .page{
	
}

.flipbook-viewport .page img{
	-webkit-touch-callout: none;
	-webkit-user-select: none;
	-khtml-user-select: none;
	-moz-user-select: none;

forms/process.php
<?php
if (($_SERVER['REQUEST_METHOD'] == 'POST') && (!empty($_POST['action']))):

	if (isset($_POST['myname'])) { $myname = $_POST['myname']; }
    if(isset($_POST['email'])){ $email = $_POST['email']; }
     

	if (isset($_POST['mycomments'])) {
			$mycomments = filter_var($_POST['mycomments'], FILTER_SANITIZE_STRING ); 
	}
	if (isset($_POST['requesttype'])) { $requesttype = $_POST['requesttype']; }

	$formerrors = false;

	if ($myname === '') :
		$err_myname = '<div class="error">Sorry, your name is a required field</div>';
		$formerrors = true;
	endif; // input field empty

	if ( !(preg_match('/[A-Za-z]+, [A-Za-z]+/', $myname)) ) :
		$err_patternmatch = '<div class="error">Sorry, the name must be in the format: Last, First</div>';
		$formerrors = true;
	endif; // pattern doesn't match
     if ($email ==='') :
        	$err_email = '<div class="error">Sorry, your email is a required field</div>';
        			$formerrors = true;
	endif; // input field empty

    // check if e-mail address is well-formed
    if (!filter_var($email, FILTER_VALIDATE_EMAIL)) {
      $emailErr = "Invalid email format";
    }
  $formdata = array (
    'myname' => $myname,
    'email' => $email,
    'mycomments' => $mycomments,    
    'requesttype' => $requesttype
  );

	if (!($formerrors)) :
		$to				= 	"mclouigene@mcdeveloper.tech";
		$subject	=		"From $myname -- Signup Page";
		$txt = "You have receied an e-mail from " .$email.".\n\n".$mycomments;
		$message	=		$mycomments;

		if (mail($to, $subject, $txt, $message)):
			$msg = "Thanks for filling out our form";
			header("location: http://test.mcdeveloper.tech/MCDad");
			exit;
		else:
			$msg = "Problem sending the message";
		endif; // mail form data

	endif; // check for form errors

endif; //form submitted
?>


includes/contact.php
<?php 
	include"process.php";
?>
<!doctype html>
<head>
<meta charset="utf-8">
	<link rel="stylesheet" href="normalize.css" />
	<link rel="stylesheet" href="mystyle.css" />
	<script src="http://code.jquery.com/jquery-1.9.1.min.js"></script>
</head>  




<?php if (isset($msg)) { echo '<div id="formmessage"><p>', $msg , '</p></div>'; } ?>
<form id="myform" name="theform" class="group" action="<?php echo $_SERVER['PHP_SELF'] ?>" method="POST">
		<span id="formerror" class="error"></span>
            <fieldset>
				<input type="text" name="myname" id="myname" title="Please enter your name" autofocus placeholder="Last, First" value="<?php if (isset($myname)) { echo $myname; } ?>" />
				<?php if (isset($err_myname)) { echo $err_myname; } ?>
				<?php if (isset($err_patternmatch)) { echo $err_patternmatch; } ?>

			</fieldset>
            <fieldest>

                	<input type="text" name="email" id="email" placeholder="Email Address" autofocus required value="<?php if(isset($email)) {echo $email; } ?>">
		<?php  if (isset($err_email)) {echo $err_email; }?>
		<?Php if (isset($err_pattternemail)) {echo $err_patternemail;}?>
            </fieldest>
			
				<div class="grouphead">Request Type</div>
				<div >
			<fieldset class="request">
						<input type="radio" name="requesttype" value="question" id="questionitem" <?php if ((isset($requesttype)) && ($requesttype==='question')) { echo "checked"; } ?>/>
						<label for="questionitem">Event Photos</label>
					</fieldset>
					<fieldset class="request">
						<input type="radio" name="requesttype" value="comment" id="commentitem" <?php if ((isset($requesttype)) && ($requesttype === 'comment')) { echo "checked"; } ?> />
						<label for="commentitem">Group Photos</label>
				</fieldset>
				<fieldset class="request">
						<input type="radio" name="requesttype" value="suggestion" id="suggestionitem" <?php if ((isset($requesttype)) && ($requesttype === 'suggestion')) { echo "checked"; } ?> />
						<label for="suggestiontem">Personel Photos</label>
				</fieldset>
				
			<fieldset class=comment>
				<!--<label for="mycomments">Comment: (For faster responses, please enter your phone number.)</label>-->
				<textarea name="mycomments" id="mycomments"><?php if (isset($mycomments)) { echo $mycomments; } ?></textarea>
		</fieldset>
		<fieldset>
		<button type="submit" name="action" value="submit">Send</button>
		</fieldset>
</form>


index.php
<!doctype html>
    <!--[if lt IE 7 ]>--> <html lang="en" class="ie6"><!-- <![endif]-->
    <!--[if IE 7 ]>    --><html lang="en" class="ie7"><!-- <![endif]-->
    <!--[if IE 8 ]>  -->  <html lang="en" class="ie8"><!-- <![endif]-->
    <!--[if IE 9 ]>  -->  <html lang="en" class="ie9"><!-- <![endif]-->
    <!--[if !IE]><!-->
<html lang="en"> <!--<![endif]-->
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
<script type="text/javascript" src=".//extras/jquery.min.1.7.js"></script>
<script type="text/javascript" src=".//extras/modernizr.2.5.3.min.js"></script>
	<link href="css/main.css" rel="stylesheet" type="text/css">
	<link href="css/basic.css" rel="stylesheet" type="text/css">

	
</head>
	
<body>
	<!-- Start of Landingpage-->
	<section class="intro">
	
		<div class="inner">
<!--	Flipbook-viewport Begin-->
<div class="flipbook-viewport">
<!--    book-zoom begin-->
  <div id="book-zoom" class="container">
<!--      Flipbook begin-->
    <div class="flipbook">
<!--        sj-hard begin-->
      <div  class="sj-hard" style="background-image:url(pages/Cover_page.02.jpg)"></div>
      <div depth="10" class="hard"></div>
<!--
        <div id="page" class="page">
        <div depth="10" class="hard"></div>
              
                  <div class="content">
                  <h4 class="leftSide">Vission 1</h4>
                    <h4 class="rightSide">1</h4>
                    
                      <h4 class="leftSide">About</h4>                      
                      <h4 class="rightSide">2</h4>
                      <h4 class="leftSide">About</h4>
                  </div>
                  
                  
        </div>
-->
      <div id="page" class="page" style="background-image:url(pages/page_0.1.jpg)">
<!--        <div class="pageNumber"> 2 </div>-->
        </div>
      <div depth="5" class="hard"></div>
      <div class="page" style="background-image:url(pages/page_1.0.jpg)">
<!--        <div class="pageNumber"> 4 </div>-->
        </div>
      <div class="hard" depth="5">
        
        </div>
      <div class="page" style="background-image:url(pages/page_2.0.jpg)">
<!--        <div class="pageNumber"> 6 </div>-->
        </div>        
        <div class="hard" depth="5"></div>
        <div class="page">
            <h1 class="headline"> Projects</h1>
            <div class="galllery">
                <a href="http://coralmulticleaning.com/" target="_blank"><img src="pages/CoralMulti.png" style="width:40%; height:50%" title="Coral Multi Cleaning " alt="Coral Multi Cleaning"></a>
                <h4> Coral Multi Cleaning Service</h4>
            </div>
            <div class="gallery"></div>
                 <a href="http://test.mcdeveloper.tech/MCDad/" target="_blank"><img src="pages/MCWeb.png" style="width:40%; height:30%" title="Apt-Art-Photography " alt="Louigene"></a>
                <h4> Mc Dad Louigene</h4>
<!--            <div class="pageNumber"> 8 </div>-->
        </div>
        <div class="hard" depth="5"></div>
        <div class="page">
            
            <h1 class="headline">Contact</h1>
            
        <section class="second">
				
				<div class="contact">
				
		<div class ="contactform">	
			<?php if (isset($msg)) { echo '<div id="formmessage"><p>', $msg , '</p></div>'; } ?>
<form id="myform" name="theform" class="group" action="<?php echo $_SERVER['PHP_SELF'] ?>" method="POST">
		<span id="formerror" class="error"></span>
            <!--<fieldset>-->
				<input type="text" name="myname" id="myname" title="Please enter your name" autofocus placeholder="Last, First" value="<?php if (isset($myname)) { echo $myname; } ?>" />
				<?php if (isset($err_myname)) { echo $err_myname; } ?>
				<?php if (isset($err_patternmatch)) { echo $err_patternmatch; } ?>

			<!--</fieldset>-->
   <!--         <fieldest>-->
                	<input type="text" name="email" id="email" placeholder="Email Address" autofocus required value="<?php if(isset($email)) {echo $email; } ?>">
		<?php  if (isset($err_email)) {echo $err_email; }?>
		<?Php if (isset($err_pattternemail)) {echo $err_patternemail;}?>
   <!--         </fieldest>-->
			<!--<fieldset class=comment>-->
				<!--<label for="mycomments">Comment: (For faster responses, please enter your phone number.)</label>-->
				<textarea name="mycomments" id="mycomments"><?php if (isset($mycomments)) { echo $mycomments; } ?></textarea>
		<!--</fieldset>-->
		<!--<fieldset>-->
		<button type="submit" name="action" value="submit">Send</button>
		<!--</fieldset>-->
</form>
				</div>
<!--                    Footer-->
                    <?php
							include 'includes/footer.html' ; 					
					?>
<!--                    <div class="pageNumber"> 10 </div>-->
					</div>	            
                		</section>

        
    
    </div>
  </div>
</div>


<script type="text/javascript">

function loadApp() {

	// Create the flipbook

	$('.flipbook').turn({
        //acceleration
			acceleration: false,
        //inclination
        inclination: 10,
			// Elevation

			elevation: 50,
			
			// Enable gradients

			gradients: true,
			//zoom
            zoom: 10,
			// Auto center this flipbook

			autoCenter: true
	});
    $("#zoom-view").bind("zoom.tap", function(event) {
	if ($(this).zoom("value")==1) {
		$(this).zoom("zoomIn", event);
	} else {
		$(this).zoom("zoomOut");
	}
});
}

// Load the HTML4 version if there's not CSS transform

yepnope({
	test : Modernizr.csstransforms,
	yep: ['.//lib/turn.js'],
	nope: ['.//lib/turn.html4.min.js'],
	both: ['.//css/basic.css'],
    both: ['.//css/main.css'],
	complete: loadApp
});
        
</script>
			</div>

	
</body>
</html>